processes/processes.tex

```diff
@@ -219,19 +252,23 @@ \subsection{Fork Functionality}
 There's no newline printed, we didn't call \keyword{fflush}, or change the buffering mode.
 The output text is therefore still in process memory waiting to be sent.
 When \keyword{fork()} is executed the entire process memory is duplicated including the buffer.
-Thus the child process starts with a non-empty output buffer which will be flushed when the program exits.
-We also say may because the contents may be unwritten given a bad program exit as well.
+Thus, the child process starts with a non-empty output buffer which may be flushed when the program exits.
+We say may because the contents may be unwritten given a bad program exit as well.
 
 To write code that is different for the parent and child process, check the return value of \keyword{fork()}.
 If \keyword{fork()} returns -1, that implies something went wrong in the process of creating a new child.
 One should check the value stored in \emph{errno} to determine what kind of error occurred.
-Commons one include \keyword{EAGAIN} and \keyword{ENOMEM} Which are essentially try again and out of memory.
-Similarly, a return value of 0 indicates that we are in the child process, while a positive integer shows that we are in parent process.
-The positive value returned by \keyword{fork()} gives as the process id (\emph{pid}) of the child.
+Common errors include \keyword{EAGAIN} and \keyword{ENOMEM} Which are essentially "try again", and "out of memory".
```

Comment:
We should remove ENOMEM because when a process uses too much memory it is usually killed by the OS.

Reply:
Replaced with ENOENT as students deal with files often in the course.